### ISSUE TITLE
Maximum Call stack size exceeded

### ISSUE THREAD

**Issue by shah1341** [OOlashyn/PCF-AttachmentGalleryControl#28]
Hi OOlashyn, first of all let me thank you for your amazing PCF control, you are AWESOME. 

we having an issue with PCF-AttachmentGalleryControl used on CRM form for Notes attachment. 
Time to time end-user reported that issue when they tried to attach a file to the Note.

**Error code 0x0: An error has occurred while trying to upload the attachment. Maximum call stack size exceeded.**

Would you please look into these to see if you can find a solution for that.

Environment: Dynamics365 on cloud.

Please feel free to ask any question to obtain more information.

P.S: I have no access to the application logs on server side. 

<img width="318" alt="new-issue" src="https://user-images.githubusercontent.com/106325691/170498441-e3204987-2ff9-48cf-8766-63466f0b276a.png">

**Comment by OOlashyn** [comment]
Hi @shah1341,
Unfortunately, the error log isn't helpful and the error described here is too generic. You can try to open console logs of the browser to check for any additional errors. Also, my control doesn't allow for note attachment. Do you mean that the user cannot attach notes when my control is on the form? Or that it cannot download the note through my control?

**Comment by shah1341** [comment]
Hi OOlashyn, thank you for reply. I agree with you that is a very generic error. I did a debug via developer tools from browser, when I do the drag and drop, console is absolutely empty and no any information logged in console. User can attach file/document to the note, no any issue here, however, time to time that error message shows up and when user try again, it goes through and file will be uploaded. 

Based on the research and reading articles from internet, it's kind of recursion is happing here, I mean one of the function called recursively until stack is full.
Attached is screenshot captured from PFC control when I did a debug.
And as I mentioned, this is not happening all the time, also I tried to reproduce it in the other environment, it's same, sometimes it appeared and most of the time works as it supposed to.

Thanks again

<img width="576" alt="debug" src="https://user-images.githubusercontent.com/106325691/170771186-b8caa7f6-5a86-4364-8d75-46603884db7f.png">
